examples/askui-nodejs-base-example/.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 dist
 .eslintcache
 report
+package-lock.json
```

Comment:
Why? Makes the installation unreproducable...

Reply:
Do you know if we use the example anywhere? Looked to me like this is not the case and we already have a lot of examples in other repos (that also look more up-to-date). So I would just remove the whole `examples` folder.

Reply:
Glad that you mentioned it. I second-guessed myself for not knowing what the `examples/askui-nodejs-base-example` is there in the first place 😄 .

I have never seen it used anywhere, so removing it will be unproblematic.